### PR TITLE
doc: update ray debugger deprecation notice

### PR DESCRIPTION
**Pull request by saihaj**
CleanShot 2024-08-14 at 22 39 44@2x

### CODE REVIEW

**Comment by can-anyscale** [comment]
not sure who is the owner of this feature, @anyscalesam do you know?

**Comment by yx367563** [comment]
But currently the distributed debugger doesn't support ray clusters on tuned clusters (kuberay), can I assume that this support will be provided in three months?

File: doc/source/ray-observability/user-guides/debug-apps/ray-debugging.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ drop into a PDB session that you can then use to:
 
 .. warning::
 
-    The Ray Debugger is an experimental feature and is currently unstable. Interfaces are subject to change.
+    The Ray Debugger is a legacy feature and will be deprecated in 3 months. You should use the :doc:`Ray Distributed Debugger <../../ray-distributed-debugger>` instead.
```
Comment (angelinalg) [inline doc/source/ray-observability/user-guides/debug-apps/ray-debugging.rst:16]:
```suggestion
    The Ray Debugger is deprecated. Use the :doc:`Ray Distributed Debugger <../../ray-distributed-debugger>` instead.
```

**Review by angelinalg — APPROVED** [review]
Just tightened up the language.

**Comment by brycehuang30** [comment]
> But currently the distributed debugger doesn't support ray clusters on tuned clusters (kuberay), can I assume that this support will be provided in three months?

Do you mean this issue https://github.com/ray-project/ray/issues/45541? We cannot guarantee the timeline but will revisit the issue if we have bandwidth. But please feel free to contribute

**Review by brycehuang30 — APPROVED** [review]

**Pull request merged**